### ISSUE TITLE
sync-cli.sh changes language

### ISSUE THREAD

**Issue by nophbert** [stwa/google_addressbook#39]
Hi,
every time i execute the sync-cli.sh (manually and cron) the language switches to 'en_US'. Is there a way to prevent this?
I'm running roundcube v. 1.1.4 stable on debian jessie.

The problem does not appear, when i am syncing through the web page.

Thanks in advance!

**Comment by stwa** [comment]
Sorry, I can't reproduce this bug, do you still get it with current stable (of roundcube)?

**Comment by nophbert** [comment]
Yes. Still the same with roundcube 1.2.0 stable (see database screenshots). Very strange behaviour.
![before](https://cloud.githubusercontent.com/assets/15978918/15815302/7a8d0392-2bcc-11e6-84ca-eef624b907c9.png)
![after](https://cloud.githubusercontent.com/assets/15978918/15815304/7efc9f8c-2bcc-11e6-9f37-1febeeb9794b.png)

**Comment by masi79** [comment]
I'm facing the same issue on my system. Any news about this?
